build/make-distribution.sh

```diff
@@ -215,8 +215,9 @@ function build_flink_client {
 if [ "$RELEASE" == "true" ]; then
   build_service
   build_spark_client -Pspark-2.4
-  build_spark_client -Pspark-3.3
-  build_flink_client -Pflink-1.14
+  build_spark_client -Pspark-3.4
+  build_flink_client -Pflink-1.15
```

Comment:
why not include flink 1.14?

Reply:
yes, it should include. 
At before, i think flink-1.15 will be backward compatible flink-1.14, so flink-1.14 can be removed. I check again that flink-1.14 is little not compatible with flink-1.15. so flink-1.14 must be included.